src/postgres/event-handlers/CommentCreated.ts

```diff
@@ -1,35 +1,39 @@
-import { CommentId } from '@subsocial/types/substrate/interfaces/subsocial';
+import { Post, PostId } from '@subsocial/types/substrate/interfaces/subsocial';
 import { substrate } from '../../substrate/subscribe';
 import { insertCommentFollower } from '../insert-follower';
 import { insertActivityComments, insertActivityForComment } from '../insert-activity';
 import { fillNotificationsWithAccountFollowers, fillNotificationsWithPostFollowers } from '../fill-activity';
 import { substrateLog as log } from '../../connections/loggers';
-import { SubstrateEvent, EventHandlerFn } from '../../substrate/types';
+import { SubstrateEvent } from '../../substrate/types';
+import { VirtualEvents } from '../../substrate/utils';
 
-export const onCommentCreated: EventHandlerFn = async (eventAction: SubstrateEvent) => {
+export const onCommentCreated = async (eventAction: SubstrateEvent, post: Post) => {
   const { data } = eventAction;
```

Comment:
Fix this copypasta on all post events handlers:

```typescript
const { data } = eventAction;
const follower = data[0].toString();
const postId = data[1] as PostId;
```

it should be something like this everywhere:

```typescript
const { author, postId } = parsePostEvent(eventAction);
```